common-lib/build.gradle

```diff
@@ -23,6 +23,8 @@ dependencies {
     compile files(ideaLib + 'jdom.jar')
 
     compile files('third_party/guava/guava-18.0.jar')
+    compile files(ideaLib + 'junit-4.11.jar')
```

Comment:
we should try to avoid adding dependencies on JUnit and Mockito to our non-test code